BringgSDK.js

```diff
@@ -668,6 +675,30 @@ var BringgSDK = (function () {
     watchingOrder = isWatching;
   };
 
+  module._setUpConfigByToken = function(developerToken) {
```

Comment:
most of the new flows in this new function are not tested (and see my comments below for possibly more branching paths).
please add tests either directly to this function or via the setCredentials as you started (and see my comments in te tests as well)